### ISSUE TITLE
Suppress: "The editor could not be opened due to an unexpected error"

### ISSUE THREAD

**Issue by edreamleo** [boltex/leointeg#217]
I get this message during startup. It goes away after the server starts. Imo, it would be smoother not to see this message.

Apparently this is a vs-code error. No part of the error message appears in leoInteg itself.

**Comment by boltex** [comment]
Yes, those opened panels stating the last body pane address can not be opened, is a remnant of the last leointeg session that vscode tries to reopen. but the leo-body filesystem has not started yet... 

The new leointeg 1.0.8 now closes those old panels upon startup. 

I'll try to make sure i remove that last opened body pane from the vscode's 'recent documents list' on the 'vscode-is-closing' event, so that it does not try to open it on startup. 

I'll comment more on this after doing some more tests and verification about that issue.
